src/geodesy.jl

```diff
@@ -69,8 +88,9 @@ An array of the same type and shape as the `marker`.
 # References
   Morphological Grayscale Reconstruction in Image Analysis: Applications and Efficient Algorithms IEEE Trans Image Process. 1993;2(2)
 """
-function underbuild(marker::AbstractArray{T,N}, mask::AbstractArray{T,N}, connectivity::AbstractArray{Bool}) where {T<:NumberLike,N}
```

Comment:
here i have restrain T to NumberLIke, because those operators don't make sens for multivariate images
Indeed there is no intuitive max/min for colors for instance

Reply:
Yes, I'm aware of this. Another alternative is to use

```julia
function underbuild(marker::AbstractArray{T1,N}, mask::AbstractArray{T2,N}, connectivity::AbstractArray{Bool}) where {T1<:NumberLike,T2<:NumberLike,N}
```

I'll add necessary type checks for non-Gray images.